### PR TITLE
Python Klein framework

### PR DESCRIPTION
**Pull request by knewmanTE**
https://github.com/twisted/klein

### CODE REVIEW

File: frameworks/Python/klein/app.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	fortunes = db_session.query(Fortune).all()
+	fortunes.append(Fortune(id=0, message="Additional fortune added at request time."))
+	fortunes.sort(key=attrgetter("message"))
+	for f in fortunes:
+		f.message = bleach.clean(f.message)
+	template = env.get_template("fortunes.html")
+	return template.render(fortunes=fortunes)
+
+if __name__ == "__main__":
+    app.run("localhost", 8080)
```
Comment (hamiltont) [inline frameworks/Python/klein/app.py]: `0.0.0.0` may be better than `localhost` here, in the event of multiple NICs

**Comment by hamiltont** [comment]
Very cool, it's going to be good to include something based on twisted

File: frameworks/Python/klein/app.py (hunk 1 above)
Comment (knewmanTE) [inline frameworks/Python/klein/app.py]: I changed `localhost` to `0.0.0.0`, bit it seems to be failing now.

**Comment by hamiltont** [comment]
Thanks!

**Pull request merged**